appveyor.yml

```diff
@@ -1,11 +1,15 @@
-image:
-  - Ubuntu
-  - Visual Studio 2015
+environment:
+  matrix:
+    - image: Ubuntu
+    - image: Visual Studio 2015
+      PYTHON_DIR: C:\\Python36
+    - image: Visual Studio 2015
+      PYTHON_DIR: C:\\Python37-x64
 
 build_script:
-  - cmd: "C:\\Python27\\python.exe -m pip install -r requirements-dev.txt"
+  - cmd: "%PYTHON_DIR%\\python.exe -m pip install -r requirements-dev.txt"
```

Comment:
here is difference. On Ubuntu the variable PYTHON_DIR is not defined so it not eval to string starting from C:\\ and this may make fail.